### DISCUSSION TITLE
Isn't it better to change the DevTools import in the document?

### DISCUSSION THREAD

**Discussion by do7be** [pmndrs/jotai#2501] [Q&A]
The `import` of Example in the hooks section of DevTools in the document is `"jotai-devtools"`.
If I import according to the example, i will also need the `UI DevTools` dependencies. (`@emotion/react` etc.)
If only want to use hooks, wouldn't it be better to write it as `"jotai-devtools/utils"`?

https://github.com/pmndrs/jotai/blob/main/docs/tools/devtools.mdx#example

If so, can I create a PR?

**Comment by dai-shi** [comment]
Yeah, probably. @arjunvegda ?

**Reply by arjunvegda** [reply]
That's a fair ask. @do7be - Please create a PR 🙏 

BTW, we're removing `@emtion/react` as a dependency so hopefully that issue should be resolved once [this PR](https://github.com/jotaijs/jotai-devtools/pull/129) is merged
